src/main/java/ch/njol/skript/sections/SecConditional.java

```diff
@@ -116,7 +134,7 @@ public boolean init(Expression<?>[] exprs,
 	@Nullable
 	@Override
 	protected TriggerItem walk(Event e) {
-		if (type == ConditionalType.ELSE || condition.check(e)) {
+		if (type == ConditionalType.ELSE || (parseIf && parseIfPassed) || condition.check(e)) {
```

Comment:
Very minor, but you only need to check that parseIfPassed is true since we already know that parseIf is true if parseIfPassed is true

Reply:
no, cause if its NOT a parseIf section, it'll be false, and no other conditions would work